samples/openapi3/client/petstore/python-experimental/tests/test_discard_unknown_properties.py

```diff
@@ -0,0 +1,129 @@
+# coding: utf-8
+
+# flake8: noqa
+
+"""
+Run the tests.
+$ docker pull swaggerapi/petstore
+$ docker run -d -e SWAGGER_HOST=http://petstore.swagger.io -e SWAGGER_BASE_PATH=/v2 -p 80:8080 swaggerapi/petstore
+$ pip install nose (optional)
+$ cd petstore_api-python
+$ nosetests -v
+"""
+from collections import namedtuple
+import json
+import os
+import re
+import shutil
+import unittest
+from six.moves.urllib.parse import urlencode, urlparse
+
+import petstore_api
+from petstore_api import Configuration, signing
+from petstore_api.rest import (
+    RESTClientObject,
+    RESTResponse
+)
+
+from petstore_api.model_utils import (
+    file_type,
+    int,
+    model_to_dict,
+    str,
+)
+
+MockResponse = namedtuple('MockResponse', 'data')
+
+class DiscardUnknownPropertiesTests(unittest.TestCase):
+
+    def test_deserialize_dog_do_not_discard_unknown_properties(self):
+        """ deserialize str, Dog) with unknown properties, strict validation is enabled """
+        config = Configuration(discard_unknown_keys=False)
+        api_client = petstore_api.ApiClient(config)
+        data = {
+            'dog': {
```

Comment:
If you want, you can change this data to just the dog dict.
That would look like:
```
            deserialized = api_client.deserialize(response,
                (petstore_api.Dog,), True)
```

Reply:
done